### ISSUE TITLE
Ukrywanie aukcji

### ISSUE THREAD

**Issue by SoftWareMG** [allegro/allegro-api#837]
Czy jest jakaś bramka REST przez którą mogę ukryć akcję zakończoną, tak by już na zawsze zniknęła z mojego panelu, mam tego sporo ponieważ jak na razie muszę ustawiać aukcje jako nieaktywna, a nie potrzebuję tych aukcji zakończonych.

![ukryj](https://user-images.githubusercontent.com/38202814/47655815-1f31da80-db8e-11e8-927e-1c1794476e0d.png)

**Comment by pawellen** [comment]
@MartaNowaczyk Czy taka opcja jest planowana?

**Comment by MartaNowaczyk** [comment]
@pawellen na ten moment nie pracujemy nad taką opcją, ale zgłoszę ją jako potrzebę do odpowiedniego zespołu. Jeśli taka opcja się pojawi poinformujemy o tym w oddzielnym wątku.

**Comment by pijablonski** [comment]
Również zgłaszam zapotrzebowanie na taką opcję.

**Comment by MartaNowaczyk** [comment]
@pijablonski dziękuję, za twój głos. Ważne by dopisywać się pod sugestią, będziemy wiedzieli wtedy ilu z was potrzebuje takiej zmiany.

**Comment by techjaw** [comment]
Też się zgłoszę a może już coś powstało? Chciałbym mieć np 5 status prócz INACTIVE,ACTIVE,ACTIVATING,ENDED np hidden/deleted, co mogłby mieć aukcje których chce się zupelnie pozbyć, ENDED zawsze mogę wznowić jak np przyjdzie mi towar na sklep

**Comment by techjaw** [comment]
A przy okazji po jakim czasie kasują się aukcje zupełnie z konta?

**Comment by MaciejFrackowiak** [comment]
@techjaw Nie, taka opcja nie pojawiła się jeszcze w REST API - Twój głos dodałem do listy. Oferty przenoszone są do archiwum po 60 dniach od dnia ich zakończenia.

**Comment by TomaszSzydlowski-sudo** [comment]
@MaciejFrackowiak Wiadmo coś na temat tej opcji?

**Comment by MaciejFrackowiak** [comment]
@TomaszSzydlowski-sudo Nie, nie mam tu na chwilę obecną żadnej aktualizacji.

**Comment by techjaw** [comment]
Miałem konta po 150tyś aukcji, pobranie wszystkich to nielada wyczyn :) A opcja byłaby super

**Comment by TomaszSzydlowski-sudo** [comment]
@techjaw Polecam serverless do pobierania aukcji, wyciągam ~ 150 aukcji / sekunde

**Comment by bugproof** [comment]
bump

**Comment by MaciejFrackowiak** [comment]
@bugproof Zmian tu żadnych nie było, nie mamy planów na dodanie takiej opcji w API.
